src/WpBookingsUiModule.php

```diff
@@ -0,0 +1,499 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Data\Container\ContainerFactoryInterface;
+use Dhii\Data\Container\ContainerGetCapableTrait;
+use Dhii\Data\Container\CreateContainerExceptionCapableTrait;
+use Dhii\Data\Container\CreateNotFoundExceptionCapableTrait;
+use Dhii\Data\Container\NormalizeKeyCapableTrait;
+use Dhii\Event\EventFactoryInterface;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\EddBookings\RestApi\Controller\ControllerInterface;
+use RebelCode\Modular\Module\AbstractBaseModule;
+use Dhii\Util\String\StringableInterface as Stringable;
+
+class WpBookingsUiModule extends AbstractBaseModule
+{
+    use ContainerGetCapableTrait;
+
+    use CreateContainerExceptionCapableTrait;
+
+    use CreateNotFoundExceptionCapableTrait;
+
+    use NormalizeKeyCapableTrait;
+
+    /**
+     * Registered booking's page ID.
+     *
+     * @var
+     */
+    public $bookingsPageId;
+
+    /**
+     * Page where metabox application should be shown.
+     *
+     * @var
+     */
+    public $metaboxPageId;
+
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param string|Stringable $key The module key.
+     * @param string[]|Stringable[] $dependencies The module  dependencies.
+     * @param ContainerFactoryInterface $configFactory The config factory.
+     * @param ContainerFactoryInterface $containerFactory The container factory.
+     * @param ContainerFactoryInterface $compContainerFactory The composite container factory.
+     * @param EventManagerInterface $eventManager The event manager.
+     * @param EventFactoryInterface $eventFactory The event factory.
+     */
+    public function __construct(
+        $key,
+        $dependencies,
+        $configFactory,
+        $containerFactory,
+        $compContainerFactory,
+        $eventManager,
+        $eventFactory
+    )
+    {
+        $this->_initModule($key, $dependencies, $configFactory, $containerFactory, $compContainerFactory);
+        $this->_initModuleEvents($eventManager, $eventFactory);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setup()
+    {
+        return $this->_setupContainer(
+            $this->_loadPhpConfigFile(WP_BOOKINGS_UI_MODULE_CONFIG_FILE),
+            [
+                'template_manager' => function ($c) {

```

Comment:
Is it possible to move service definitions into a different place? Maybe just a separate PHP file?

Reply:
Like a secondary config file?

Reply:
Yeaz, totally.

Reply:
Maybe, no need to use Config implementation for it. Just simply move.